resources/assets/js/vendor.js

```diff
@@ -10,6 +10,13 @@ require('./forms');
 require('./sticky-headers');
 require('./moment-countdown');
 
+moment.updateLocale('en', {
```

Comment:
Shouldn't `const moment = require('moment');` be used first?

Reply:
`require('imports-loader?this=>window&define=>false&exports=>false!moment');` makes `moment` globally available.

Reply:
Hm, ok, TIL ;)